app/src/backend/mocks/createCollection.ts

```diff
@@ -0,0 +1,55 @@
+import firebaseApp from "../../firebase";
```

Comment:
List still doesn't refresh everytime when we create a collection. Is there any difficulty here?

Reply:
Weird, with emulator works properly, but in local it doesn't!